tests/e2e/Services/Projects/ProjectsCustomServerTest.php

```diff
@@ -34,25 +33,5 @@ public function testCreateProjectRule()
         $response = $this->client->call(Client::METHOD_DELETE, '/proxy/rules/' . $response['body']['$id'], $headers);
 
         $this->assertEquals(204, $response['headers']['status-code']);
-
-        // prevent functions domain
-        $functionsDomain = System::getEnv('_APP_DOMAIN_FUNCTIONS', '');
-
-        $response = $this->client->call(Client::METHOD_POST, '/proxy/rules', $headers, [
-            'resourceType' => 'api',
-            'domain' => $functionsDomain,
-        ]);
-
-        $this->assertEquals(400, $response['headers']['status-code']);
-
-        // prevent sites domain
-        $sitesDomain = System::getEnv('_APP_DOMAIN_SITES', '');
-
-        $response = $this->client->call(Client::METHOD_POST, '/proxy/rules', $headers, [
-            'resourceType' => 'api',
-            'domain' => $sitesDomain,
-        ]);
-
-        $this->assertEquals(400, $response['headers']['status-code']);
```

Comment:
Instead of removing test, we can add assurance that status code is 200, since it's allowed now

Reply:
Yes, we should still keep the tests, however, they should still fail as site and function domains aren't still allowed